### ISSUE TITLE
nvidia-settings issue while selecting refresh rate on a rotated display.

### ISSUE THREAD

**Issue by vide0hanz** [NVIDIA/nvidia-settings#57]
Came across this bug on my system while adding a secondary monitor in portrait mode, it appears to be specific to how the nvidia-settings gui interprets refresh rate settings - both of my monitors have a native refresh rate of 165 and it was a bit of a pain to figure out how to set my rotated monitor to its native refresh rate, until I discovered how these inconsistencies worked. 

The nvidia settings appears to default to the resolution to "auto" when setting the rotation via X Server Display Configuration. You can then select the native resolution and refresh rate, but the representation for the display in the X Screen viewport gets switched to landscape mode. However, once you select a **different** refresh rate, the orientation switches back to portrait mode. Here are some example screenshots:

**STEP 1**
Open `nvidia-settings` and rotate the display you wish to rotate, the resolution defaults to "auto" and refresh rate dropdown menu is greyed out, as expected:

![2020-11-16_11-55](https://user-images.githubusercontent.com/59800033/99290397-da744d00-2803-11eb-9e0e-5fa9a3a6662b.png)

**STEP 2**
Select the native resolution in order to set the desired refresh rate, but take note that the **first** rate you select causes the monitors representation in the overall X Screen layout to revert to landscape mode:

![incorrect](https://user-images.githubusercontent.com/59800033/99290543-098abe80-2804-11eb-82fe-c65c2fe1fdba.png)

**STEP 3**
Now, select the **actual** refresh rate you want to use, this will correct these discrepancies:

![correct](https://user-images.githubusercontent.com/59800033/99290649-2fb05e80-2804-11eb-915d-3a9da0bf9411.png)

**SUMMARY**
What I ended up having to do was rotate my display and set the native resolution, then I had to an undesired refresh rate **before** selecting my desired one for it to work properly. 

I believe the expected behavior is to choose resolution, orientation, and refresh rate correctly the first time. 

Let me know if there are any additional details you'd like me to provide.

**Comment by lewiji** [comment]
This bug has existed for well over a year; I'm surprised no one has acknowledged or addressed this issue, it's super annoying

**Comment by vide0hanz** [comment]
Right. While it did not take me long to figure out what was happening and work around it, it was very frustrating. This post was mostly written in the spirit of helping others who may be running into this.

**Comment by lewiji** [comment]
Yeah, after dealing with it for the 50th time I googled it and was surprised to find only this issue, still open, with no replies or acknowledgement! I guess it's a bit of a niche edge case, we can't be the only ones with portrait, high refresh rate monitors on Linux, right? :)

**Comment by svenstaro** [comment]
@aaronp24 any chance you could take a look at this? It's really quite annoying.

**Comment by alkaris2** [comment]
My issue with nvidia-settings is that it keeps auto-selecting Auto for Refresh Rate, instead of the defined max refresh rate that my monitor supports which is at 144Hz, because every reboot it chooses to select 60Hz refresh rate under Auto and I find it extremely annoying I have to set it to the max refresh rate every time I startup my computer.

**Comment by vide0hanz** [comment]
> My issue with nvidia-settings is that it keeps auto-selecting Auto for Refresh Rate, instead of the defined max refresh rate that my monitor supports which is at 144Hz, because every reboot it chooses to select 60Hz refresh rate under Auto and I find it extremely annoying I have to set it to the max refresh rate every time I startup my computer.

 I know it seems obvious, but are you running `nvidia-settings` as root and saving the X configuration file? This will ensure an xorg.conf file generated by nvidia-settings is created which is responsible for settings persisting after rebooting.
